.github/actions/verify-dotnet-consolidation/action.yml

```diff
@@ -22,8 +26,8 @@ runs:
       uses: Lombiq/GitHub-Actions/.github/actions/install-dotnet-tool@dev
       with:
         name: dotnet-consolidate
-        version: 2.0.0
+        version: 4.1.0
     - name: Verify consolidation status
       shell: pwsh
       working-directory: ${{ inputs.directory }}
-      run: dotnet consolidate -s (Get-ChildItem ${{ inputs.solution-path }}).FullName
+      run: dotnet consolidate -s (Get-ChildItem ${{ inputs.solution-path }}).FullName --excludedVersionsRegex ${{ inputs.exclude-version-regex }}
```

Comment:
Are you sure this will work with more complex expressions? The `${{ }}` just substitutes, so I think it will break with anything containing shell operators such as `*`. I'd rather go for
```
env:
  ExcludeVersionRegex: ${{ inputs.exclude-version-regex }}
```

Reply:
Yeah, good point.